scss/utilities/_spacing.scss

```diff
@@ -1,3 +1,7 @@
+// Width
+
+.w-100 { width: 100% !important; }
```

Comment:
The current name parallels the naming of the other utilities and sets a pattern that additional width utilities can follow (e.g. `.w-50`).
Also, `.w-100` indicates exactly what the class does (`width: 100%`, and nothing else) (albeit utilizing an abbreviation), whereas `.full-width` could potentially be interpreted differently (e.g. `max-width: 100%`, `display: block`) by readers lacking context.